AndelaOSP/art-dashboard
# #159229841 Fix field values selection for modals

## What does this PR do?
Fix field values selection for modals

## Description of Task to be completed?
```
After the changes on API endpoint pagination we need to update the way we're fetching data for 
 modals.

To review:

Adding Sub-category
Adding Asset Make
Adding Model Number
Adding Asset Type
Currently we're looping through the responses (confirm this is the current implementation for all the above).

Going forward we should use url parameter paginated=false to return all items (non-paginated)
```
## How should this be manually tested?
```
Once logged in, on the assets page, click on the + icons on either Sub-category, Asset Make,  Model Number or  Asset Type. 

Each modal displayed has a dropdown field which should list all the existing data.
```
## What are the relevant pivotal tracker stories?
#159229841

## Any background context you want to add?
N/A
## Important notes
N/A
## Packages installed
N/A
## Deployment note
N/A
## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
<img width="762" alt="screen shot 2018-07-25 at 10 56 54" src="https://user-images.githubusercontent.com/27014080/43187234-95140cf4-8ff9-11e8-9add-d375f2957754.png">
<img width="769" alt="screen shot 2018-07-25 at 10 57 11" src="https://user-images.githubusercontent.com/27014080/43187236-954335a6-8ff9-11e8-99c7-b714addf60ce.png">
<img width="752" alt="screen shot 2018-07-25 at 10 57 21" src="https://user-images.githubusercontent.com/27014080/43187237-956cc614-8ff9-11e8-9e90-5b81011bf18d.png">
<img width="799" alt="screen shot 2018-07-25 at 10 57 30" src="https://user-images.githubusercontent.com/27014080/43187238-9595a430-8ff9-11e8-910b-cfa32d3d1e50.png">
